frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -323,13 +323,47 @@ function ReaderRolling:addToMainMenu(menu_items)
                 end
             end
         },
+        {
+            text_func = function()
+                return T(_("Number of lines: %1"), G_reader_settings:readSetting("copt_overlap_lines") or 1)
+            end,
+            enabled_func = function()
+                return self.show_overlap_enable
+            end,
+            callback = function(touchmenu_instance)
+                local SpinWidget = require("ui/widget/spinwidget")
+                UIManager:show(SpinWidget:new{
+                    width = Screen:getWidth() * 0.75,
+                    value = G_reader_settings:readSetting("copt_overlap_lines") or 1,
+                    value_min = 1,
+                    value_max = 10,
+                    precision = "%d",
+                    ok_text = _("Set"),
+                    title_text =  _("Set overlapped lines"),
+                    text = _([[
+When page overlap is enabled, some lines from the previous pages are shown on the next page.
+You can set how many lines are to be shown.]]),
+                    callback = function(spin)
+                        G_reader_settings:saveSetting("copt_overlap_lines", spin.value)
+                        touchmenu_instance:updateItems()
+                    end,
+                })
+            end,
+            keep_menu_open = true,
+            help_text = _([[
+When page overlap is enabled, some lines from the previous pages are shown on the next page.
+You can set how many lines are to be shown.]]),
+            separator = true,
+        },
     }
-    for _, menu_entry in ipairs(self.view:genOverlapStyleMenu()) do
+    local overlap_enabled_func = function() return self.show_overlap_enable end
+    for _, menu_entry in ipairs(self.view:genOverlapStyleMenu(overlap_enabled_func)) do
         table.insert(page_overlap_menu, menu_entry)
     end
     menu_items.page_overlap = {
         text = _("Page overlap"),
         enabled_func = function() return self.view.view_mode ~= "page" end,
+        help_text = _([[When page overlap is enabled, some lines from the previous pages are shown on the next page.]]),
```

Comment:
```suggestion
        help_text = _([[When page overlap is enabled, some lines from the previous pages will be shown on the next page.]]),
```

Just making the suggestions consistent.